client/web/src/cody/management/subscription/new/CodyProCheckoutForm.tsx

```diff
@@ -17,76 +17,130 @@ import {
     Icon,
     Input,
     Label,
-    LoadingSpinner,
     H3,
-    useDebounce,
+    LoadingSpinner,
 } from '@sourcegraph/wildcard'
 
 import { CodyAlert } from '../../../components/CodyAlert'
-
-import { PayButton } from './PayButton'
+import { useCreateTeam } from '../../api/react-query/subscriptions'
+import { StripeAddressElement } from '../StripeAddressElement'
+import { StripeCardDetails } from '../StripeCardDetails'
 
 import styles from './NewCodyProSubscriptionPage.module.scss'
 
 const MIN_SEAT_COUNT = 1
 const MAX_SEAT_COUNT = 50
 
-export const CodyProCheckoutForm: React.FunctionComponent<{
-    creatingTeam: boolean
+interface CodyProCheckoutFormProps {
+    initialSeatCount: number
     customerEmail: string | undefined
-}> = ({ creatingTeam, customerEmail }) => {
+}
+
+export const CodyProCheckoutForm: React.FunctionComponent<CodyProCheckoutFormProps> = ({
+    initialSeatCount,
+    customerEmail,
+}) => {
+    const stripe = useStripe()
+    const elements = useElements()
     const navigate = useNavigate()
 
-    const { total, lineItems, updateLineItemQuantity, email, updateEmail, status } = useCustomCheckout()
+    const creatingTeamMutation = initialSeatCount > 1
```

Comment:
nit: "creating" sounds like pending process to me. 
```suggestion
    const shouldCreateTeamMutation = initialSeatCount > 1
```

Reply:
Thanks, was bad naming on my part. I ended up going with just `isTeam`: https://github.com/sourcegraph/sourcegraph/pull/63213/commits/a09ff2f7da2083af0c5f59af4a9b9e6f61dba32e